### ISSUE TITLE
Console in/output Windows incorrect

### ISSUE THREAD

**Issue by ronaldbarendse** [maxcnunes/slack-files#1]
The console output on Windows 10 (build 1803) contains lots of unknown symbols (bacause of coloring?) and the input isn't correctly accepted (bacause of different newlines, `\r\n` instead of `\n`):

```
Would you like to delete those files?
  [36m1)[0m No. Stop here.
  [36m2)[0m Yes. Delete all them.
  [36m3)[0m Yes. But ask me to confirm each one of them.
=> 2
Stopping without delete any file.
```

**Comment by maxclaus** [comment]
Yep, that is because of the color. Sorry about those issues I tested it only on Unix based systems.
Appreciate if you could send a PR for that 🤞 😄

**Comment by ronaldbarendse** [comment]
I installed Windows Subsystem for Linux and used the Linux build from the Ubuntu bash instead: that worked perfectly 👍

I don't have any prior experience with Go and just needed a tool for a one-off job 👼 Thanks for creating this though! 🥇
